pancancer_evaluation/utilities/analysis_utilities.py

```diff
@@ -466,8 +466,8 @@ def generate_nonzero_coefficients_lasso_range_msi(results_dir):
         # use negative indexing since some feature selection methods
         # have underscores in them - indexing from the back of the
         # filename is safer/more consistent
-        seed = int(coefs_file.split('_')[-4].replace('s', ''))
-        lasso_param = coefs_file.split('_')[-2].replace('c', '')
+        seed = int(coefs_file.split('_')[-5].replace('s', ''))
```

Comment:
Will this mean you have problems when you try to process files generated under the old system?

Reply:
I shouldn't - I ran a bash script to fix all the old ones. It was pretty simple so I didn't track it in this repo. If I missed any I think I'd rather just fix them than add an "if" statement to handle that case.